.fixtures.yml

```diff
@@ -1,6 +0,0 @@
----
-fixtures:
-  repositories:
-    stdlib: 'https://github.com/puppetlabs/puppetlabs-stdlib.git'
```

Comment:
We don't have a dependency on stdlib and the cron symlink is automatically created by recent versions of puppetlabs-spec-helper.